modules/apps/fragment/fragment-impl/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "fragment-impl",
+	"scripts": {
+		"build": "liferay-npm-scripts build",
+		"checkFormat": "liferay-npm-scripts check",
+		"format": "liferay-npm-scripts fix",
+		"test": "liferay-npm-scripts test",
```

Comment:
Maybe we don't need this?

Reply:
:thinking: We need at least the build... Unless we move the package.json to src/main/resources/META-INF/resources which is the expected place. 

However, even if the bundle has no .js files inside, I would treat it as a regular bundler-enabled project because in the end it will have a JSBundle, JSPackage and lots of JSModules. Also, we could even need to attach some .js files for auxiliary functionality in the future...

Reply:
Note: Related to [this](https://github.com/liferay-frontend/liferay-portal/pull/268#discussion_r474459514)